cypress/e2e/hub/approvals.cy.ts

```diff
@@ -1,12 +1,46 @@
 import { Approvals } from './constants';
+import { randomString } from '../../../framework/utils/random-string';
 
 describe('Approvals', () => {
+  let thisCollectionName: string;
+  let namespace: string;
+
   before(() => {
     cy.hubLogin();
+    cy.getNamespace('ibm');
+    cy.navigateTo('hub', Approvals.url);
+  });
+
+  after(() => {
+    cy.deleteCollection(thisCollectionName, 'ibm', 'published');
   });
 
   it('it should render the approvals page', () => {
+    cy.verifyPageTitle(Approvals.title);
+  });
+
+  it('user can upload a new collection and approve it', () => {
+    thisCollectionName = 'test' + randomString(4).toLowerCase();
```

Comment:
Maybe the string can be little longer, for example 5? Just to be sure its not the same as other collection. The name of any entity should start with 'e2e' string - we discussed it in monday, this would help us identifying our testing data and batch deleting old data in script outside of tests.